core/db/default/spree/zones.rb

```diff
@@ -3,6 +3,9 @@
 south_america = Spree::Zone.where(name: 'South America', description: 'South America', kind: 'country').first_or_create!
 middle_east = Spree::Zone.where(name: 'Middle East', description: 'Middle East', kind: 'country').first_or_create!
 asia = Spree::Zone.where(name: 'Asia', description: 'Asia', kind: 'country').first_or_create!
+no_limits = Spree::Zone.where(name: 'No Limits', description: 'Includes all the countries of the world!', kind: 'country').first_or_create!
```

Comment:
@przemosk let's not create this zone altogether please

Reply:
@damianlegawiec if we resign from default checkout zone for store, should we allow `Zone` dropdown be blank ? Atm, always "default" will be `EU_VAT`  

<img width="1164" alt="Zrzut ekranu 2020-10-30 o 10 29 17" src="https://user-images.githubusercontent.com/1455599/97688700-c954dd80-1a9a-11eb-912d-f23e359ed4b0.png">

Reply:
@przemosk let's allow it to be blank, we can set the `No Limits` string for the blank option 

```
include_blank: 'No limits'
```